README.md

```diff
@@ -10,38 +10,35 @@ To run the API you will need appropriate AWS credentials. See the [Wiki](https:/
 
 ### pycurl
 
-pycurl needs to be installed separately, with some specific compiler flags and steps. You'll need to follow these steps
-in any repo where you wish to run celery locally. The following steps have been adapted from https://gist.github.com/vidakDK/de86d751751b355ed3b26d69ecdbdb99
 
+First install make sure you have some prerequisite dependencies installed:
```

Comment:
Think this sentence has a typo/grammar issue

Reply:
Oops, fixed 👍.